Makefile

```diff
@@ -274,12 +280,30 @@ showVariables:
 all: showVariables lint debug release apache testdebug testrelease fixrights
 
 .PHONY: user
-user:
+user: env
 	source $(USER_SOURCE) && make all
 
 .PHONY: build
 build: showVariables .build-artefacts/devlibs .build-artefacts/requirements.timestamp $(SRC_JS_FILES) debug release
 
+
+.PHONY: .build-artefacts/nvm-version
+.build-artefacts/nvm-version: .build-artefacts/last-nvm-version
+ifneq ($(LAST_NVM_VERSION),$(NVM_VERSION))
```

Comment:
We should only install it when it's not already installed. So check first if nvm command exists.